lib/ansible/modules/packaging/language/pip.py

```diff
@@ -235,6 +237,61 @@
                              'pip': 'import pkg_resources; print(pkg_resources.get_distribution("pip").version)'}
 
 
+class Package(Requirement):
+    def __init__(self, name_string, version_string=None):
+        if version_string is not None:
+            if version_string[0].isdigit():
+                name_string = "%s ==%s" % (name_string, version_string)
+            else:
+                name_string = "%s %s" % (name_string, version_string)
+        super(Package, self).__init__(name_string)
+
+    def have_version_specifier(self):
+        if len(self.specs) == 0:
+            return False
+        return True
+
+    def get_name(self):
+        return self.name.lower()
+
+    def satisfied_by(self, version_to_test):
+        return self.specifier.contains(version_to_test)
+
+    def install_command(self):
+        return str(self)
+
+
+def _is_valid_pkg_name(name):
+    try:
+        Requirement(name)
+        return True
+    except RequirementParseError:
+        return False
```

Comment:
please note that behavior for non-caught exceptions is to return `None`, so please add another `return False` in the end and maybe replace this one with `pass` or a docstrinig with the explanation.